src/authn/index.js

```diff
@@ -788,11 +795,12 @@ async function renewTokens(refreshToken) {
       throw new Error(`failed to renew tokens: ${response.status} ${response.statusText}: ${body}`);
   }
 
-  const {id_token: idToken, access_token: accessToken} = JSON.parse(body);
+  const {id_token: idToken, access_token: accessToken, refresh_token: refreshToken_} = JSON.parse(body);
 
   return {
     idToken,
     accessToken,
+    refreshToken: refreshToken_ ?? refreshToken,
```

Comment:
_(non-blocking)_

Suggestion: Use a more meaningful variable name, e.g.

```suggestion
    refreshToken: refreshTokenFromIdP ?? refreshToken,
```

Reply:
Ah, ha, I briefly dithered on names for that before choosing the trailing `_` form (as `x_` is fairly conventional for "`x`, but modified"). Seemed ok to me as declaration and usage were so close together, but yeah, relies on connotations rather than being descriptive on its own.

I've changed it to `refreshTokenUpdate`.